### ISSUE TITLE
Slow startup of Zettlr when Zettlr website was down. Is there any explanation for that correlation?

### ISSUE THREAD

**Issue by nathanlesage** [Zettlr/Zettlr#3333]
### Discussed in https://github.com/Zettlr/Zettlr/discussions/3332

<div type='discussions-op-text'>

<sup>Originally posted by **fmartinp** March 20, 2022</sup>
Hello,

I have noticed that during the downtime of Zettlr website on 20/03/2022, Zettlr software was taking a significant more amount of time (several seconds) to load. When Zettlr website was up again, Zettlr's startup time when back to nominal.

The automatic update was disabled in my configuration, so I do not understand this correlation between the Zettlr website downtime and the slowness of the startup of the software if there is no other connection between Zettlr host and the software.

I am using the Zettlr 2.2.4 for Windows x64 (Zettlr-2.2.4-x64.exe)

Could anyone have an explanation for that correlation of the slow startup and the website downtime?

Thanks in advance.</div>

**Basically what we need to see is if any of the API accesses by the app are blocking. We need to *not* await the Promises of accessing the API during boot.**

**Comment by nathanlesage** [comment]
I have literally no idea why it took me so long to fix that, but I recently fixed that.
